t/Makefile.am

```diff
@@ -294,6 +294,8 @@ dist_check_SCRIPTS = \
 	issues/t4184-sched-simple-restart.sh \
 	issues/t4222-kvs-assume-empty-dir.sh \
 	issues/t4331-job-manager-purged-events.sh \
+	issues/t4378-content-flush-force.sh \
```

Comment:
Commit msg: consider adding a commit body and re-wording commit subject to remove reference to github issue numbers. Not a requirement, but I do find that github issue numbers make a nice supplement to the commit message, but aren't as nice when they are the sole context. (E.g. when browsing `git log` locally you will have to separately look up issues online)

E.g. something like this (I may have gotten description wrong)
```
testsuite: add regression tests for content flush issues

Problem: No tests in the testsuite ensure that content flush is forced after
`flux content flush` nor that any dirty cache entries are flushed after a content
backing module is loaded. (Issues #4378 and #4379)

Add regression tests for these specific issues.
```

As I said, just a suggestion and it may be more personal preference, so feel free to ignore.